apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/McsTransformer.java

```diff
@@ -149,7 +149,8 @@ void addDiagnosisCodes(McsClaim.Builder claimBuilder, Parser.Data<String> data,
                                   .map(
                                       dxVersionCode -> {
                                         try {
-                                          return Integer.parseInt(dxVersionCode);
+                                          // Convert ("9", "0") literals to (0, 1) enum values
+                                          return (Integer.parseInt(dxVersionCode) + 1) % 10;
```

Comment:
neat, but wondering if something like this reads a bit better?
```
return Map.of("9", 0, "0", 1).get(dxVersionCode)
```

Reply:
I can add a map to the class instead